### PR TITLE
feat: [SFEQS-953] add `io-sign` web sdk

### PR DESCRIPTION
**Pull request by lucacavallaro**
#### List of Changes

Add `@io-sign/web-sdk` package that contains the `io-sign` element that powers the CTA flow

#### Motivation and Context

This package exports the `<io-sign>` web component that can be embedded into issuers websites to show QRCode with IO links or redirect mobile user directly to the IO app

#### Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: packages/web-sdk/src/components/Skeleton.ts [hunk 1]
```diff
@@ -0,0 +1,42 @@
+import { LitElement, html, css } from "lit";
```
Comment (gunzip) [inline packages/web-sdk/src/components/Skeleton.ts:1]: may you add a comment on each component about its purpose?

File: packages/web-sdk/src/io-sign.ts [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+  @state()
+  showQrCode = false;
+
+  @provide({ context: IOLinkProviderContext })
+  @property({ attribute: false })
+  IOLinkProvider = new IOSignLinkProvider();
+
+  theme: HTMLLinkElement | null = null;
+
+  connectedCallback() {
```
Comment (gunzip) [inline packages/web-sdk/src/io-sign.ts:50]: what's the purpose of this method? maybe a comment would help

File: packages/web-sdk/public/demo.html [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Firma con IO demo</title>
+    <script type="module" src="./io-sign.js"></script>
+  </head>
+  <body>
+    <io-sign id="cta"></io-sign>
+    <script>
+      const cta = document.getElementById("cta");
```
[the remark this answers is not in the document]
Comment (lucacavallaro) [reply packages/web-sdk/public/demo.html:12]: they are just development files (useful for local testing)

File: packages/web-sdk/src/io-sign.ts [hunk 4]
```diff
@@ -0,0 +1,141 @@
...
+  showQrCode = false;
+
+  @provide({ context: IOLinkProviderContext })
+  @property({ attribute: false })
+  IOLinkProvider = new IOSignLinkProvider();
+
+  theme: HTMLLinkElement | null = null;
+
+  // Due to Web Component limitations, it's not possible to
+  // delcare custom fonts inside the component-scoped CSS
```
Comment (gunzip) [inline packages/web-sdk/src/io-sign.ts:51]:
```suggestion
  // declare custom fonts inside the component-scoped CSS
```

**Review by gunzip — APPROVED** [review]

File: packages/web-sdk/src/io-sign.ts (hunk 4 above)
Comment (lucacavallaro) [reply packages/web-sdk/src/io-sign.ts]: thankz

**Review by silvicir — APPROVED** [review]

**Review by grausof — APPROVED** [review]
LGTM

**Pull request merged**